packages/veritone-react-common/src/components/TranslationEngineOutput/index.js

```diff
@@ -0,0 +1,171 @@
+import React, { Component } from 'react';
+import { arrayOf, shape, number, string, func } from 'prop-types';
+import classNames from 'classnames';
+
+import Select from 'material-ui/Select';
+import { MenuItem } from 'material-ui/Menu';
+
+import EngineOutputHeader from '../EngineOutputHeader';
+import TranslationContent from './TranslationContent';
+import styles from './styles.scss';
+
+export default class TranslationEngineOutput extends Component {
+  static propTypes = {
+    title: string,
+    contents: arrayOf(
+      shape({
+        startTimeMs: number,
+        stopTimeMs: number,
+        series: arrayOf(
+          shape({
+            startTimeMs: number,
+            stopTimeMs: number,
+            status: string,
+            words: arrayOf(
```

Comment:
Has this been confirmed that translation text comes in words[], but not in object.text ?